probe-rs/src/probe/cmsisdap/mod.rs

```diff
@@ -413,7 +405,7 @@ impl DebugProbe for DAPLink {
     }
 
     fn get_name(&self) -> &str {
-        "DAPLink"
+        "CMSISDAP"
```

Comment:
I would prefer CMSIS-DAP here, I think it's commonly written that way.

Reply:
Thanks, good idea. Updated.